contrib/packaging/rpm/cfg/cilium.spec.envsubst

```diff
@@ -0,0 +1,70 @@
+%global commit0 ${COMMIT}
+%global shortcommit0 %(c=%{commit0}; echo ${c:0:7})
+
+Name:          cilium
+Version:       ${VERSION}
+Release:       0.git%{shortcommit0}%{?dist}
+Summary:       BPF & XDP for containers
+License:       Apache
+URL:           https://github/cilium/cilium
+ExclusiveArch: x86_64
+Source0:       https://github.com/cilium/%{name}/archive/%{commit0}.tar.gz#/%{name}-%{shortcommit0}.tar.gz
+BuildRequires: golang < 1.8.0, glibc-devel(x86-32)
+Requires:      docker-engine >= 1.12, clang >= 3.8, glibc-devel(x86-32), iproute >= 4.6, kernel >= 4.8
```

Comment:
Currently there is no packages for clang 3.8 in official Fedora repo, they have been updated to version 3.9. You need to install the old ones using archive links:
```
dnf install https://kojipkgs.fedoraproject.org//packages/clang/3.8.1/1.fc25/x86_64/clang-3.8.1-1.fc25.x86_64.rpm \
    https://kojipkgs.fedoraproject.org//packages/clang/3.8.1/1.fc25/x86_64/clang-libs-3.8.1-1.fc25.x86_64.rpm \
    https://kojipkgs.fedoraproject.org//packages/llvm/3.8.1/2.fc25/x86_64/llvm-libs-3.8.1-2.fc25.x86_64.rpm \
    https://kojipkgs.fedoraproject.org//packages/compiler-rt/3.8.1/1.fc25/x86_64/compiler-rt-3.8.1-1.fc25.x86_64.rpm
```
We will have to wait for additional packages for 3.8 (similarly to versions 3.4, 3.5, 3.7). When they will pop up in repo I'll update requirements from `clang` to `clang3.8`.